yt_dlp/extractor/easybroadcast.py

```diff
@@ -0,0 +1,91 @@
+
+from yt_dlp.extractor.common import InfoExtractor
+from yt_dlp.utils import (
+    ExtractorError,
+)
+
+
+class EasyBroadcastLiveIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:\w+\.)?player\.easybroadcast\.io/events/(?P<id>[0-9a-zA-Z_]+)'
+    _EMBED_REGEX = [rf'<iframe [^>]*\bsrc=[\'"](?P<url>{_VALID_URL})']
+    _TESTS = [
+        {
+            'url': 'https://al24.player.easybroadcast.io/events/66_al24_u4yga6h',
+            'info_dict': {
+                'id': '66_al24_u4yga6h',
+                'title': r're:Al24 \d{4}-\d{2}-\d{2} \d{2}:\d{2}',
+                'ext': 'mp4',
+                'live_status': 'is_live',
+            },
+            'params': {
+                'nocheckcertificate': True,
+            },
+        },
+        {
```

Comment:
- remove unnecessary line breaks and fix indentation
- skip download test for live content
- change `title` to `str` since it may vary

please apply these to the other tests in this PR and your other PR
```suggestion
    _TESTS = [{
        'url': 'https://al24.player.easybroadcast.io/events/66_al24_u4yga6h',
        'info_dict': {
            'id': '66_al24_u4yga6h',
            'ext': 'mp4',
            'title': str,
            'live_status': 'is_live',
        },
        'params': {
            'nocheckcertificate': True,
            'skip_download': 'Livestream',
        },
    }, {
```

Reply:
check suggested changes again and simply add `skip_download` to `params`